addons/content-index/src/main/java/org/commonjava/indy/content/index/IndexedStorePathTransformer.java

```diff
@@ -0,0 +1,55 @@
+package org.commonjava.indy.content.index;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.infinispan.query.Transformer;
+
+import java.io.IOException;
+
+/**
+ * A customized infinispan {@link org.infinispan.query.Transformer} used for {@link org.commonjava.indy.content.index.IndexedStorePath}
+ * to support it to be used as infinispan cache key in indexing.
+ */
+public class IndexedStorePathTransformer
+        implements Transformer
+{
+    //FIXME: not sure why the IndyObjectMapper cannot be successfully injected here, so use default objmapper instead
+    //    @Inject
+    //    private IndyObjectMapper objectMapper;
```

Comment:
I'm guessing this transformer isn't initialized by CDI. Many of these sorts of binding helper classes in different frameworks are just instantiated using reflection or something like that...I'd guess this is similar.